README.md

```diff
@@ -86,6 +86,17 @@ sudo make install
 
 `runc` will be installed to `/usr/local/sbin/runc` on your system.
 
+#### Version string customization
+
+You can see the runc version by running `runc --version`. You can append a custom string to the
+version using the `VERSION_EXTRA` make variable when building, e.g.:
+
+```bash
+make VERSION_EXTRA="+build-1"
+```
+
+Bear in mind to include some separator for readability.
```

Comment:
Thanks for bringing up all nightmares of correct version strings (I think Debian expects `-` to be used for packaging revision, and `~` for other bits, but it may differ between the version string and package name).

I guess not our concern, and a fun exercise for the user 🙈

Reply:
Yeah, leaving it open for others seems the simplest option :)